### DISCUSSION TITLE
Upload database

### DISCUSSION THREAD

**Discussion by jcubic** [sqlitecloud/discussions#8] [Help] [answered]
Is there a way to upload an existing SQLite database file?

**Comment by marcobambini** [comment, marked as answer]
Sure @jcubic, select Upload database icon in the Databases section
![Screenshot 2024-05-07 at 22 17 38](https://github.com/sqlitecloud/discussions/assets/11838145/521c7aac-27bc-488c-9b75-0ee1fbfb6137)
